.github/dependabot.yml

```diff
@@ -5,3 +5,8 @@ updates:
     directory: "/"
     schedule:
       interval: "daily"
+  - package-ecosystem: "docker"
+    open-pull-requests-limit: 20
+    directory: "/"
```

Comment:
I'm not sure if `/` will detect `model-server/Dockerfile`.
Maybe `directory` needs to be `/model-server`.

I guess we can just merge in try it out.

Reply:
I was wondering the same thing. But there currently is [no way to test this before merging](https://github.com/dependabot/dependabot-core/issues/4605), so we'll have to see.